bp_fe/src/include/bp_fe_icache_stat_info.vh

```diff
@@ -0,0 +1,21 @@
+/**
+ *  Name:
+ *    bp_fe_icache_stat_info.vh
+ *
+ *  Description:
+ *    stat_mem entry format.
+ */
+
+`ifndef BP_FE_ICACHE_STAT_INFO_VH
+`define BP_FE_ICACHE_STAT_INFO_VH
+
+`define declare_bp_fe_icache_stat_info_s(ways_mp)  \
```

Comment:
Isn't this part of the cache_service_if?

Reply:
It wasn't, but now I have added it.

Reply:
Nice!